byron/ledger/executable-spec/test/DoctestDiscover.hs

```diff
@@ -1,12 +1 @@
-{-# LANGUAGE CPP #-}
-
-#if !MIN_VERSION_GLASGOW_HASKELL(8,10,0,0) || MIN_VERSION_GLASGOW_HASKELL(8,11,0,0)
 {-# OPTIONS_GHC -F -pgmF doctest-discover #-}
-#else
```

Comment:
AIUI this has not been fixed?

Reply:
Yep. This has not been fixed.  We can `cabal-docspec` instead and delete these files altogether.

Reply:
I have pushed a commit to delete these files.